<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ClusterControlBase</class>
 <widget class="QMainWindow" name="ClusterControlBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Infiscape Cluster Control</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QFrame" name="mToolbox" >
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="mStack" >
        <widget class="QWidget" name="page" />
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionReload" />
    <addaction name="actionNew" />
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
    <addaction name="actionSave_As" />
    <addaction name="separator" />
    <addaction name="action_Exit" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>680</y>
     <width>778</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionReload" />
   <addaction name="actionNew" />
  </widget>
  <widget class="QDockWidget" name="mStatusWindow" >
   <property name="allowedAreas" >
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>9</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget" >
       <widget class="QWidget" name="tab" >
        <attribute name="title" >
         <string>Tab 1</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_2" >
        <attribute name="title" >
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionReload" >
   <property name="text" >
    <string>&amp;Reload</string>
   </property>
  </action>
  <action name="actionNew" >
   <property name="text" >
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="text" >
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As" >
   <property name="text" >
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="action_Exit" >
   <property name="text" >
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_About" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
